eng/pipelines/corefx-base.yml

```diff
@@ -0,0 +1,123 @@
+  jobs: []
+
+  # EACH JOB SHOULD INCLUDE THE FOLLOWING PROPERTIES (ASIDE FROM THE REQUIRED ONES IN THE JOB SCHEMA)
+
+  # Required: as part of the strategy matrix, the following variables should be defined
+  #     _configuration: Debug | Release
+  #     _architecture: x64 | x86 | arm | arm64
+  #     _framework: (netcoreapp, netfx, uap, etc).
+  #     _helixQueues: Windows.Amd64 (Only needed if submitToHelix -> true.) -- Queues should be separated by + if multiple.
+
+  # Required: as part of the variables object, the following values need to be passed:
+  #     _skipTests: true | false
+  #     _outerloop: true | false
+
+  # Required: submitToHelix -> Boolean -> Value to know if it should submit tests payloads to helix.
+  
+  # Optional: buildScriptPrefix -> String -> string to append to Unix build script.
+  #   buildScriptPrefix: 'HOME=/home/ ' -> 'HOME=/home/ ./build.sh ...'
+
+  # Optional: buildExtraArguments -> String -> string to append at the end of the build arguments
+  #   buildExtraArguments: /p:RuntimeOS=win10 -> 'build.cmd ... /p:RuntimeOS=win10'
+
+  # Optional: customBuildSteps -> Array -> list of steps to run instead of the common build steps.
+  #   customBuildSteps:
+  #     - script: echo Hello World
+  #       displayName: MyScript
+
+  # Optional: enableMicrobuildPlugin -> Boolean - if microbuild plugin for signing should be enabled
+  # Default: false
+  
+  # Optional: preBuildSteps -> Array -> list of steps to be executed before common build steps.
+  # In example, to install build dependencies, or setup an environment.
+  #   preBuildSteps:
+  #     - script: brew install cmake
+  #       displayName: Install Cmake
+
+  # timeoutInMinutes: if a job doesn't define it, we will default it to 150 minutes
+
+jobs:
+  - ${{ each job in parameters.jobs }}:
+    - template: ../common/templates/job/job.yml
+      parameters:
+        variables:
+        # pass along job variables
+        - ${{ each variable in job.variables }}:
+          - ${{ if ne(variable.name, '') }}:
+            - name: ${{ variable.name }}
+              value: ${{ variable.value }}
+          - ${{ if ne(variable.group, '') }}:
+            - group: ${{ variable.group }}
+          
+        # Define OS specific commands
+        - ${{ if eq(parameters.targetOS, 'Windows_NT') }}:
+          - name: _buildScript
+            value: build.cmd -ci
+          - name: _msbuildCommand
+            value: powershell -ExecutionPolicy ByPass -NoProfile eng\common\msbuild.ps1 -warnaserror:0 -ci
+        - ${{ if ne(parameters.targetOS, 'Windows_NT') }}:
+          - name: _buildScript
+            value: ${{ job.buildScriptPrefix }}./build.sh --ci
+          - name: _msbuildCommand
+            value: ${{ job.buildScriptPrefix }}./eng/common/msbuild.sh --warnaserror false --ci
+
+        - ${{ if eq(job.enableMicrobuild, 'true') }}:
+          - name: _TeamName
+            value: DotNetCore
+          - name: _SignType
+            value: true
```

Comment:
Shouldn't this be "Real"?

Reply:
Oh yeah. However I’m not yet enabling micro build anywhere. Will update once the build is done. Good catch!